Dockerfile

```diff
@@ -372,4 +397,6 @@ EXPOSE 8080
 
 ENTRYPOINT ["/usr/local/bin/dumb-init", "--", "/entrypoint.sh"]
 
+VOLUME /var/lib/docker
```

Comment:
I'll add a comment, but essentially, for docker-in-docker to work: `The Dockerfile also indicates that /var/lib/docker should be a volume. This is important, because the filesystem of a container is an AUFS mountpoint, composed of multiple branches; and those branches have to be “normal” filesystems (i.e. not AUFS mountpoints). In other words, /var/lib/docker, the place where Docker stores its containers, **cannot be an AUFS filesystem**. Therefore, we instruct Docker that this path should be a volume. Volumes have many purposes, but in this scenario, we use them as a pass-through to the “normal” filesystem of the host machine. The /var/lib/docker directory of the nested Docker will live somewhere in /var/lib/docker/volumes on the host system.

Reference: https://blog.docker.com/2013/09/docker-can-now-run-within-docker/

Reply:
Explanation added.

Reply:
As explained in further comment - I think we do not have to do this and we can run dind image separately rather than running the docker image inside airflow-testing.